core/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ dependencies {
   implementation(libs.commons.io)
   implementation(libs.commons.lang3)
   implementation(libs.guava)
+  implementation(libs.h2db)
```

Comment:
You need to add this to the LICENSE.bin.

Reply:
Added